Edit doc overall based on PyAnsys doc style guidelines (#655)
pyansys/pyfluent: doc/source/api/core/solver/events.rst

=============

An instance of ``EventsManager`` exists as an active ``events_manager`` property in each

> solution-mode session object. It allows for registering the client callback with server-side

Reviewer: It's not really *the* callback -  the original should have been pluralised. Also, we're not allowing _for_ something, we're simply allowing it. The meaning is different. However, the original grammar is incorrect. 

**You can register callbacks with the EventsManager, which are triggered by server-side events.**

> events. Whenever an event occurs, the registered callback is triggered with session ID and event

Reviewer: So, actually, here the occurrence of events is mentioned. So we can simplify the above, and rewrite the whole thing.

> solution-mode session object. It allows for registering the client callback with server-side
> events. Whenever an event occurs, the registered callback is triggered with session ID and event
> information passed as arguments to the callback. It is useful in solution monitoring and updating graphics 
> during runtime.

Reviewer: ```suggestion
solution-mode session object. You can register client callbacks with the EventsManager. Whenever a server event occurs, the registered callback, which receives session ID and event
information arguments to the callback. The EventsManager is useful for solution monitoring and updating graphics .
```
